charts/camunda-platform/charts/optimize/templates/deployment.yaml

```diff
@@ -0,0 +1,100 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "optimize.fullname" . }}
+  labels: {{- include "optimize.labels" . | nindent 4 }}
+  annotations: {{- toYaml  .Values.global.annotations | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "optimize.matchLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels: 
+        {{- include "optimize.labels" . | nindent 8 }}
+        {{- if .Values.podLabels }}
+        {{- toYaml .Values.podLabels | nindent 8 }}
+        {{- end }}
+    spec:
+      containers:
+      - name: {{ .Chart.Name }}
+        {{- if .Values.image.tag }}
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        {{- else }}
+        image: "{{ .Values.image.repository }}:{{ .Values.global.image.tag }}"
+        {{- end }}
+        env:
+          - name: CAMUNDA_OPTIMIZE_ZEEBE_ENABLED
+            value: "true"
+          - name: OPTIMIZE_ELASTICSEARCH_HOST
+            value: {{ .Values.global.elasticsearch.host | quote }}
+          - name: OPTIMIZE_ELASTICSEARCH_HTTP_PORT
+            value: {{ .Values.global.elasticsearch.port | quote }}
+          - name: SPRING_PROFILES_ACTIVE
+            value: "ccsm"
+          - name: CAMUNDA_OPTIMIZE_IDENTITY_ISSUER_URL
+            value: {{ .Values.global.identity.auth.publicIssuerUrl | quote }}
+          - name: CAMUNDA_OPTIMIZE_IDENTITY_ISSUER_BACKEND_URL
+            value: "http://{{ include "common.names.dependency.fullname" (dict "chartName" "keycloak" "chartValues" . "context" $) | trunc 20 | trimSuffix "-" }}:80/auth/realms/camunda-platform"
+          - name: CAMUNDA_OPTIMIZE_IDENTITY_CLIENTID
```

Comment:
@RomanJRW it seems that Optimize only supports `CLIENTID` the other tools, like tasklist and operate use (or support) `CLIENT_ID` Is this expected? I stumbled over it, because it didn't worked without the right settings here.

Reply:
Thanks for the heads up. Once JIRA wakes up, I'll create a ticket for us to change this in future 👍